airbyte/sources/base.py

```diff
@@ -100,12 +101,28 @@ def set_streams(self, streams: list[str]) -> None:
         )
         self.select_streams(streams)
 
+    def _log_warning_preselected_stream(self, streams: str | list[str]) -> None:
+        """Logs a warning message indicating stream selection which are not selected yet"""
+        if streams == "*":
+            print(
+                "Warning: Config is not set yet. All streams will be selected after config is set."
+            )
+        else:
+            print(
+                f"Warning: Config is not set yet. {streams} streams will be selected after config is set."
+            )
```

Comment:
Line length can't be auto-fixed by the linter...

```suggestion
            print(
                "Warning: Config is not set yet. "
                f"Streams to be selected after config is set: {streams}"
            )
```